uber/tests/models/test_attendee.py

```diff
@@ -104,7 +104,7 @@ def test_is_transferrable(monkeypatch):
     assert Attendee(paid=c.PAID_BY_GROUP).is_transferable
     assert not Attendee(paid=c.NOT_PAID).is_transferable
 
-    assert not Attendee(paid=c.HAS_PAID, trusted=True).is_transferable
+    assert not Attendee(paid=c.HAS_PAID, trusted=True).is_transferable # TODO
```

Comment:
You seem to have a bunch of blank ``TODO`` comments without actually explaining what it is you need to do.  This is like tying a string around your finger to remind yourself of something instead of writing yourself a note explaining what you need to remember :P